tools/license/check-license.sh

```diff
@@ -4,21 +4,30 @@ if [ ! -f "Package.swift" ]; then
 	echo "\`check-license.sh\` must be run in repository root folder: \`./tools/license/check-license.sh\`"; exit 1
 fi
 
+IFS=$'\n'
+
 function files {
 	find -E . \
-		-iregex '.*\.(swift|h|m)$' \
+		-iregex '.*\.(swift|h|m|py)$' \
 		-type f \( ! -name "Package.swift" \) \
 		-not -path "*/.build/*" \
 		-not -path "*Pods*" \
-		-print0	
+		-not -path "*Carthage/Build/*" \
+		-not -path "*Carthage/Checkouts/*"
 }
 
-files | while IFS= read -r -d '' file; do
-  if ! grep -q "Apache License Version 2.0" "$file"
-  then
-  	echo "No license in $file"
-  	exit 1
-  fi
```

Comment:
`then` was never executed somehow because of `while` above?

Reply:
It was executed, but the `exit 1` was not exiting the entire process. I'm not the bash expert, but I think the `exit 1` was only exiting the `while's` scope as there's a pipe `|` operator used, which seems to create subprocess 🤷. Anyway, now it works and we can get feedback on multiple broken files, not only the first one.